src/librustc_mir/interpret/validity.rs

```diff
@@ -776,14 +781,14 @@ impl<'rt, 'mir, 'tcx, M: Machine<'mir, 'tcx>> ValueVisitor<'mir, 'tcx, M>
                         // For some errors we might be able to provide extra information
                         match err.kind {
                             err_ub!(InvalidUndefBytes(Some(ptr))) => {
```

Comment:
should these error variants get renamed, too then?

Reply:
Well, yes -- and `ScalarMaybeUndef` and `UndefMask` and a few other things. But for now I was focusing on the user-visible parts.

Reply:
I opened an issue for that: https://github.com/rust-lang/rust/issues/71193